Fix relays

evanshultz opened the pull request:
I noticed a couple minor issues with some relay footprints.

Finder 30.22
https://www.finder-relais.net/en/finder-relays-series-30.pdf
Before (left fab line was not vertical):
![image](https://user-images.githubusercontent.com/1936989/77337703-7c2fb280-6ce6-11ea-9b98-28604a037ef7.png)
After:
![image](https://user-images.githubusercontent.com/1936989/77337642-66ba8880-6ce6-11ea-9233-80c0310ce7fc.png)

I didn't rotate this one since the pins names didn't include `1` and I'm fixing the issue I found and plan to stop at that.

Omron G5V-2
https://omronfs.omron.com/en_US/ecb/products/pdf/en-g5v2.pdf
Before (line sticking down at the bottom left, but I also noticed that the body dimensions and a few pins weren't in the right place and I moved the text while I was at it):
![image](https://user-images.githubusercontent.com/1936989/77337821-ac775100-6ce6-11ea-9354-8d62689f12ce.png)
After:
![image](https://user-images.githubusercontent.com/1936989/77337493-32df6300-6ce6-11ea-9f17-0c4e00f70e4f.png)
![image](https://user-images.githubusercontent.com/1936989/77337972-e7798480-6ce6-11ea-9124-f435ddb486f2.png)


---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [ ] Provide a URL to a datasheet for the footprint(s) you are contributing
- [ ] An example screenshot image is very helpful 
- [ ] If there are matching symbol or 3D model pull requests, provide link(s) as appropriate
- [ ] Check the output of the Travis automated check scripts - fix any errors as required
- [ ] Give a reason behind any intentional library convention rule violation.

---

Be patient, we maintainers are volunteers with limited time and need to check your contribution against the datasheet. You can speed up the process by providing all the necessary information (see above). And you can speed up the process even more by providing a dimensioned drawing of your contribution. A tutorial on how to do that is found here: https://forum.kicad.info/t/how-to-check-footprint-correctness/9279 (This is optional!)

chschlue commented on Relay_THT.pretty/Relay_DPDT_Finder_30.22.kicad_mod:
```diff
@@ -17,8 +17,8 @@
+  (fp_line (start -1.26 -0.2) (end -1.26 -8.86) (layer F.Fab) (width 0.12))
+  (fp_line (start -1.26 -8.86) (end 19.04 -8.86) (layer F.Fab) (width 0.12))
   (fp_line (start 19.04 -8.86) (end 19.04 1.24) (layer F.Fab) (width 0.12))
   (fp_line (start 19.04 1.24) (end 0.04 1.2) (layer F.Fab) (width 0.12))
```
This line is also not horizontal.

evanshultz commented:
Thanks for looking!

Right you are. Thank you. I fixed that and moved the bevel to exactly 1mm since the fab lines no longer closed. But that revealed the fab lines were 0.12mm so I slimmed them all down to 0.1mm. Then the silk obviously wasn't evenly offset from the fab lines, so I made them all 0.11mm away.
![image](https://user-images.githubusercontent.com/1936989/79701243-5b5c6d80-8250-11ea-81b2-fd605c8e6b25.png)

I also corrected the fab line width on teh G5V-2 footprint as well.

chschlue commented:
Thanks.
